contactless-support.html

```diff
@@ -0,0 +1,113 @@
+---
+layout: skinny
+title: "Contactless Payments on Public Transportation: Interest Form for Cal-ITP Support"
+class_name: no-banner
+form:
+  form_target: form-contactless-support
+  confirm_target: form-confirm
+modes:
+  - Bus
+  - Commuter Coach
+  - Dial-A-Ride / Paratransit
+  - Light Rail
+  - Commuter Rail
+  - Bikeshare
+  - Other
+---
+
+<p>
+  Thank you for your interest in the California Integrated Travel Project (Cal-ITP; <a href="https://www.calitp.org"
+    target="_blank">www.calitp.org</a>), an initiative of the California State Transportation Agency (CalSTA) and
+  California Department of Transportation (Caltrans) to enable easy and accessible travel planning and payments.
+</p>
+
+<p>
+  Whether you represent a public Transit Provider based in California or elsewhere in the U.S., Cal-ITP can provide
+  materials and support to help you navigate modernizing your fleet to accept open-loop EMV fare payments from riders'
+  contactless credit/debit cards and mobile wallets.
+</p>
+
+<p>
+  To help us better assist you, we ask that you complete this form indicating your interest in contactless payment
+  systems and the State of California's Master Service Agreements (MSAs), established by the CA Department of General
+  Services (DGS) for Payment Acceptance Devices (PADs, aka fare validators) and Transit Processor Services (fare
+  calculation software).
+</p>
+
+<p>
+  With this information, we can determine how best to support you with developing a project scope, engaging with
+  vendors, and implementing your new fare payment system.
+</p>
+
+<p>
+  If you have any questions as you complete this form, please either leave them in the Comments / Queries box at the end
+  of this form or reach out to us directly: <a href="mailto:hello@calitp.org" target="_blank">hello@calitp.org</a>
+</p>
+
+<p class="indicates-required">
+  <em>{% include forms/required-label.html %} Indicates required fields</em>
+</p>
+
+{% assign honeypot = "_hp_" %}
+{% assign name = page.form.form_target | replace_first: 'form-', '' %}
+<form data-netlify="true" netlify-honeypot="{{ honeypot }}" id="{{ page.form.form_target }}" method="POST" name="{{ name }}">
+  <section>
+    <h3>Section 1 of 4: Transit Provider Information</h3>
```

Comment:
```suggestion
    <h2 class="h3">Section 1 of 4: Transit Provider Information</h2>
```

Make sure the headings go in order, otherwise you'll end up with a broken document map; e.g.

![image](https://user-images.githubusercontent.com/1246453/157824150-a7ba0094-5886-49b6-bf9d-4dba9a561935.png)

_(Source: HeadingsMap extension ([links for each browser are at the bottom](https://rumoroso.bitbucket.io/)))_

If you want the `h3` font size, then use the `h3` class on an `h2` tag (confusing, I know, but semantically correct lol). Same for the other headings in this section.

Reply:
Thanks for keeping me semantically honest 😉